src/components/network/selection-creation-panel/contingency-filter-creation/contingency-filter-creation-fields.tsx

```diff
@@ -4,9 +4,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-import { ElementType, FILTER_EQUIPMENTS, FormEquipment, SelectInput } from '@gridsuite/commons-ui';
+import {
+    CONTINGENCY_LIST_EQUIPMENTS,
+    ElementType,
+    FILTER_EQUIPMENTS,
+    FormEquipment,
+    SelectInput,
+} from '@gridsuite/commons-ui';
 import { Grid } from '@mui/material';
-import { EQUIPMENT_TYPES, equipmentTypeToLabel } from 'components/utils/equipment-types';
```

Comment:
seems like it was the only use of `equipmentTypeToLabel` then remove it as well ?